crates/astria-conductor/src/executor.rs

```diff
@@ -210,16 +226,19 @@ impl<C: ExecutionClient> Executor<C> {
             return Ok(None);
         }
 
-        if let Some(execution_hash) = self.sequencer_hash_to_execution_hash.get(&block.block_hash) {
+        if let Some(execution_block) = self
+            .sequencer_hash_to_execution_block
+            .get(&block.block_hash)
+        {
             debug!(
                 height = block.header.height.value(),
-                execution_hash = hex::encode(execution_hash),
+                execution_hash = hex::encode(execution_block.clone().hash),
```

Comment:
can you just clone the hash instead of the block?

Reply:
I think cloning in general is unnecessary. tendermint's `Hash` impls `AsRef<[u8]>` and `hex::encode` is defined for `T: AsRef<[u8]>`

Reply:
`execution_block.hash` is a `Vec<u8>` here

Reply:
Doesn't change the argument tho: https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-AsRef%3C%5BT%5D%3E-for-Vec%3CT,+A%3E :-)